shared/profile/index.desktop.js

```diff
@@ -625,11 +632,18 @@ const styleShowcasedTeamAvatar = {
 }
 
 const styleShowcasedTeamName = {
-  ...globalStyles.flexBoxRow,
+  ...Styles.globalStyles.flexBoxRow,
   alignItems: 'center',
   justifyContent: 'center',
   alignSelf: 'center',
-  paddingLeft: globalMargins.tiny,
+  paddingLeft: Styles.globalMargins.tiny,
 }
 
+const styles = Styles.styleSheetCreate({
```

Comment:
I didn't want to get too into an entire style refactor of the Profile screen so it seemed like the best action was to just create a new `styles` object for the `FloatingMenu` style.